driver-core/src/test/unit/com/mongodb/operation/OperationUnitSpecification.groovy

```diff
@@ -44,6 +44,28 @@ import java.util.concurrent.TimeUnit
 
 class OperationUnitSpecification extends Specification {
 
+    // Have to add to this map for every server release
+    private static final SERVER_TO_WIRE_VERSION_MAP = [
+            [2, 6]: 2,
+            [3, 0]: 3,
+            [3, 2]: 4,
+            [3, 4]: 5,
+            [3, 6]: 6,
+            [4, 0]: 7,
+            [4, 1]: 8
+    ]
+
+    static int getMaxWireVersion(List<Integer> serverVersion) {
```

Comment:
This doesn't really belong here because it's used by OperationFunctionalSpecification, but it is only unit tests in functional that actually use it, so...